pyproject.toml

```diff
@@ -32,10 +32,13 @@ python = "^3.6"
 #
 # For example, the version constraint of python package `Pillow` should be `^5`.
 Pillow = "^5"
+dataclasses = {version = "^0.8", python = "< 3.7"}
 
 [tool.poetry.dev-dependencies]
 Sphinx = "^3.5.3"
-pyproject-indirect-import-detector = "^0.1.1"
+cryptography = "^3.4.7"
+pyOpenSSL = "^20.0.1"
+pyproject-indirect-import-detector = "^0.1.2"
```

Comment:
Because we need https://github.com/kenoss/pyproject-indirect-import-detector/pull/18 .